src/components/newdapp/step2/index.js

```diff
@@ -38,10 +38,13 @@ GridLayout.protoTypes = {
 }
 
 const TemplateLayout = ({ image, name, description, selected, onTemplateSelected } = props) => (
-    <div onClick={onTemplateSelected} class={classNames([style.templateLayout], { [style.selected]: selected })}>
+    <div onClick={onTemplateSelected} class={classNames([style.templateLayout], { [style.selected]: selected })}
+         style={{alignItems:'center', justifyContent:'center', flexDirection: 'column', display:'grid'}}>
```

Comment:
@waheed25 lets try leaving all styles out of the html and use the css style instead.